pkgs/top-level/all-packages.nix

```diff
@@ -5581,6 +5581,21 @@ with pkgs;
 
   gcc_multi = wrapCCMulti gcc;
 
+  llvm-multi = llvmPkgs:
+    if system == "x86_64-linux" then
+      callPackages ../development/compilers/llvm/multi.nix {
+        gcc32 = pkgsi686Linux.gcc;
+        gcc64 = pkgs.gcc;
+        llvmPackages = llvmPkgs;
+      }
+    else throw "Multilib ${cc.name} not supported on '${system}'";
+
+  gccMultiStdenv = overrideCC stdenv gcc_multi;
+  clangMultiStdenv = llvmMultiPackages.stdenv;
+  llvmMultiPackages_4 = llvm-multi llvmPackages_4;
+  llvmMultiPackages_5 = llvm-multi llvmPackages_5;
```

Comment:
I don't think you should wrap `llvmPackages_4` and `llvmPackages_5`, they will be unused and untested.

Reply:
Okay.  We could perhaps alternatively test these as well (at least using same basic test added in this PR), but maybe it's better to wait until someone requests this or there are in-tree uses.